pkg/types/config.go

```diff
@@ -41,6 +41,7 @@ type PlatformConfig struct {
 	Kubernetes            Kubernetes          `yaml:"kubernetes" json:"kubernetes,omitempty"`
 	Kpack                 Kpack               `yaml:"kpack,omitempty" json:"kpack,omitempty"`
 	Ldap                  *Ldap               `yaml:"ldap,omitempty" json:"ldap,omitempty"`
+	Letsencrypt           LetsencryptIssuer   `yaml:"letsencrypt,omitempty" json:"letsencrypt,omitempty"`
```

Comment:
this should probably come under certmanager, as it doesn't make sense on its own